doc/sphinx/language/extensions/match.rst

```diff
@@ -290,6 +290,43 @@ This example emphasizes what the printing settings offer.
 
        Print snd.
 
+Conventions about unused pattern-matching variables
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Pattern-matching variables unused in the right-hand side of ``=>`` is
+considered to be the sign of a potential error. For instance, it could
```

Comment:
```suggestion
Pattern-matching variables that are not used on the right-hand side of ``=>`` are
considered the sign of a potential error. For instance, it could
```

Reply:
Thanks! (And sorry for the "is".)